_dev/eslint-plugin-fxa/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "eslint-plugin-fxa",
+  "version": "2.0.2",
+  "description": "eslint plugin for Firefox Accounts",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {
+    "eslint": "6.6.0",
```

Comment:
Latest eslint is v8.36.0. 
I assume dependabot will check the _dev/ directory now as well, correct?
I didn't see anywhere obvious in https://github.com/mozilla/fxa/blob/main/.github/dependabot.yml where we're scoping updates to packages/** directory, but I think this is the first package.json file outside of root and packages/, per:

```sh
git ls-files | grep "package.json"

_dev/eslint-plugin-fxa/package.json
package.json
packages/123done/package.json
packages/browserid-verifier/package.json
packages/db-migrations/package.json
packages/fortress/package.json
packages/functional-tests/package.json
packages/fxa-admin-panel/package.json
packages/fxa-admin-server/package.json
packages/fxa-auth-client/package.json
packages/fxa-auth-server/package.json
packages/fxa-content-server/package.json
packages/fxa-customs-server/package.json
packages/fxa-dev-launcher/package.json
packages/fxa-event-broker/package.json
packages/fxa-geodb/package.json
packages/fxa-graphql-api/package.json
packages/fxa-payments-server/package.json
packages/fxa-profile-server/package.json
packages/fxa-react/package.json
packages/fxa-settings/package.json
packages/fxa-shared/package.json
```

Reply:
> I assume dependabot will check the _dev/ directory now as well, correct?

Good question.  I don't know for certain.  I don't think there's a way to explicitly exclude directories, so I guess that means it just scans everything.